x/gov/tally.go

```diff
@@ -26,7 +26,7 @@ func tally(ctx sdk.Context, keeper Keeper, proposal Proposal) (passes bool, tall
 	keeper.vs.IterateBondedValidatorsByPower(ctx, func(index int64, validator sdk.Validator) (stop bool) {
 		currValidators[validator.GetOperator().String()] = validatorGovInfo{
 			Address:         validator.GetOperator(),
-			Power:           sdk.NewDecFromInt(validator.GetPower()),
+			Power:           sdk.NewDec(validator.GetTendermintPower()),
```

Comment:
We should instead use `GetTokens` here, no `x/gov` code should be dependent on TM power conversions

Reply:
doen